clearwater-etcd/usr/share/clearwater/bin/poll_etcd_cluster.sh

```diff
@@ -57,7 +57,13 @@ cluster_state()
     # cluster error).
     local out=`nice -n 19 etcdctl cluster-health 2> /dev/null`
     if [ "$?" = 0 ] ; then
-      local cluster_state_regex="^cluster is unhealthy$"
+      local unhealthy_cluster_state_regex="^cluster is unhealthy$"
+      if [[ $out =~ $cluster_state_regex ]]
+      then
+        return 2
```

Comment:
Both cases represent the cluster being unhealthy - I think they want the same error code then